src/features/quality-stats/QualityStatsCollector.js

```diff
@@ -210,14 +250,20 @@ class QualityStatsCollector {
 
         // Get the collected data associated with this PC.
         const pcData = this._getPcData(pc);
-        const { transport: { rtts } } = pcData;
+        const { transport: { rtts }, inboundVideoExperiences } = pcData;
```

Comment:
what sets the `inboundVideoExperiences` data?

Reply:
There is a function called `_updateInboundVideoExperience ` called from the loop that iterates the reports in the stats entry.